lib/widgets/subscription_list.dart

```diff
@@ -45,11 +45,35 @@ class _SubscriptionListPageBodyState extends State<SubscriptionListPageBody> wit
     list.sort((a, b) {
       if (a.isMuted && !b.isMuted) return 1;
       if (!a.isMuted && b.isMuted) return -1;
+      
+      final isEmojiA = _startsWithEmoji(a.name);
+      final isEmojiB = _startsWithEmoji(b.name);
+      if (isEmojiA != isEmojiB) {
+        return isEmojiA ? -1 : 1;
+      }
```

Comment:
nit:

```suggestion
      final aStartsWithEmoji = /* … */
      final bStartsWithEmoji = /* … */
      if (aStartsWithEmoji && !bStartsWithEmoji) return -1;
      if (!aStartsWithEmoji && bStartsWithEmoji) return 1;
```

More accurate name ("starts with", not "is"), and more parallel to the logic above:

```dart
      if (a.isMuted && !b.isMuted) return 1;
      if (!a.isMuted && b.isMuted) return -1;
```

Reply:
Let's also leave a comment like:

```dart
      // A user gave feedback wanting zulip-flutter to match web in putting
      // emoji-prefixed channels first; see #1202.
      // For matching web's ordering completely, see:
      //   https://github.com/zulip/zulip-flutter/issues/1165
```

Reply:
Hmm, the code must be consistent. I will keep this in mind. Thanks!